dvc/commands/queue/kill.py

```diff
@@ -11,20 +11,31 @@ class CmdQueueKill(CmdBase):
     """Kill exp task in queue."""
 
     def run(self):
-        self.repo.experiments.celery_queue.kill(revs=self.args.task)
+        self.repo.experiments.celery_queue.kill(
+            revs=self.args.task, force=self.args.force
+        )
 
         return 0
 
 
 def add_parser(queue_subparsers, parent_parser):
-    QUEUE_KILL_HELP = "Kill actively running experiment queue tasks."
+    QUEUE_KILL_HELP = "Send SIGINT(Ctrl-C) to running experiment queue tasks."
     queue_kill_parser = queue_subparsers.add_parser(
         "kill",
         parents=[parent_parser],
         description=append_doc_link(QUEUE_KILL_HELP, "queue/kill"),
         help=QUEUE_KILL_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+    queue_kill_parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        default=False,
+        help="Send SIGKILL (kill -9) instead to running experiment queue "
+        "tasks. (The default `queue kill` will terminate more gracefully,"
+        " collecting and cleaning up all resources)",
+    )
```

Comment:
Maybe something like

`Forcefully and immediately kill running experiment queue tasks`